Makefile

```diff
@@ -8,3 +8,7 @@ OPENSHIFT_NAMESPACES =
 .PHONY: $(shell test -f common/common.mk || echo >&2 'Please do "git submodule update --init" first.')
 
 include common/common.mk
+
+.PHONY: test-upstream-unittests
```

Comment:
I would suggest the name `test-upstream` w/o unittests

Reply:
Sure.

Reply:
Fixed with [f42a446](https://github.com/sclorg/s2i-nodejs-container/pull/415/commits/f42a446d1d17b54cb90d77c13bfb363092e20835).